src/tests/ftest/launch.py

```diff
@@ -2240,6 +2243,31 @@ def indent_text(indent, text):
     return " " * indent + str(text)
 
 
+def reset_server_storage(test_file, args):
+    """Reset the server storage for the hosts that ran servers in the test.
+
+    This is a workaround to enable binding devices back to nvme or vfio-pci after they are unbound
+    from vfio-pci to nvme.  This should resolve the "NVMe not found" error seen when attempting to
+    start daos engines in the test.
+
+    Args:
+        test_file (dict): a dictionary of the test script/yaml file
+        args (argparse.Namespace): command line arguments for this program
+
+    Returns:
+        int: status code: 0 = success, 512 = failure
```

Comment:
Looks like the only return is `return 0`

Reply:
That's intended.  If we ever expand this function to return a failure it should return 512, hence the comment.